routing/message/handlers/passthrough/command_handler.go

```diff
@@ -25,34 +25,30 @@ const commandHandlerName = "passthrough_command_handler"
 
 // A simple command passthrough handler that forwards all cloud-to-device messages from Azure IoT Hub to local MQTT broker on a preconfigured topic.
 type commandHandler struct {
-	topics string
+	topic string
 }
 
 // CreateCommandHandler instantiates a new command handler that forward cloud-to-device messages to the local message broker using the given topic.
-func CreateCommandHandler() handlers.MessageHandler {
-	return &commandHandler{}
+func CreateCommandHandler(topic string) handlers.CommandHandler {
+	return &commandHandler{
+		topic: topic,
+	}
 }
 
-// Init gets the local topic to publish the messages.
-func (h *commandHandler) Init(settings *config.AzureSettings, connSettings *config.AzureConnectionSettings) error {
-	h.topics = settings.PassthroughCommandTopic
+// Init does nothing.
+func (h *commandHandler) Init(connInfo *config.CloudConnectionInfo) error {
```

Comment:
In the end, it's Azure API, do we need such empty method if there is no use case for the command handler? Do you think that it can be used by another implementation? Any use case/idea?

Reply:
This simple implementation here makes no use of the remote connection properties. 
But another one, e.g. command handler implementation of [#21] should make use of DeviceID, HubName... e.g. using the given properties to construct a local topic for Hono/Ditto protocols.